cmd/melt/main.go

```diff
@@ -238,3 +255,30 @@ func completeColor(truecolor, ansi256, ansi string) string {
 	}
 	return ansi
 }
+
+// setLanguage sets the language of the big39 mnemonic seed.
+func setLanguage(language string) error {
+	switch strings.ToLower(language) {
+	case "chinese-simplified":
```

Comment:
You can also match against `zh` here, the ISO two letter code for simplified Chinese

Reply:
Oh, missed that one. Will add it tomorrow. Is there also an iso code for traditional Chinese? :thinking:

Reply:
Not as far as I know, but put some suggestions in my other comment.